File: src/main/java/de/hinundhergestellt/jhuh/vendors/ready2order/model/ProductsIdPutRequestMixin.java
package de.hinundhergestellt.jhuh.vendors.ready2order.model;

import com.fasterxml.jackson.annotation.JsonIgnore;

@SuppressWarnings("unused")
public abstract class ProductsIdPutRequestMixin {

    @JsonIgnore
    public abstract Integer getProductVatId();
}

File: src/main/java/de/hinundhergestellt/jhuh/vendors/ready2order/openapi/model/ProductsIdPutRequestMixin.kt
package de.hinundhergestellt.jhuh.vendors.ready2order.openapi.model

import com.fasterxml.jackson.annotation.JsonIgnore

@Suppress("unused")
abstract class ProductsIdPutRequestMixin {

    @get:JsonIgnore
    abstract val productVatId: Int?

    @get:JsonIgnore
    abstract val productgroupId: Int?
}
